apps/ui/src/views/PersonView.vue

```diff
@@ -0,0 +1,214 @@
+<template>
+  <div>
+    <v-card class="person-card">
+      <v-card-text>
+        <div v-if="personData.loading.value && personData.result.value" class="loading">
+          <v-progress-circular size="80" indeterminate color="red-lighten-1" />
+          <p>Loading...</p>
+        </div>
+        <div v-else class="details-container">
+          <v-avatar :size="200" :image="personDetails?.profilePicture?.path ?? RPITV_Logo" />
+          <h1>{{ personDetails?.name }}</h1>
+          <footer>{{ personDetails?.pronouns }}</footer>
+          <p class="description">{{ personDetails?.description }}</p>
+          <v-dialog v-if="personImageData.result.value?.personImageCount" v-model="isPersonImageDialogOpened" width="1100">
+            <template #activator="{ props: activatorProps }">
+              <v-btn class="mt-7" @click="imageDetails = []" v-bind="activatorProps">View Photos</v-btn>
+            </template>
+            <template #default>
+              <PersonImageCard :personId="route.params.id as string" />
+              <v-btn class="position-fixed right-0 mx-3 my-1" variant="text" icon="fal fa-xmark" @click="isPersonImageDialogOpened = false" />
+            </template>
+          </v-dialog>
+          <h2 class="mt-5">Club Positions</h2>
+          <div class="roles">
+            <div class="text-center" v-for="role in personDetails?.roles">
+              <h3 style="color: #ff6363">{{ role.role?.name }}</h3>
+              <h4>{{ moment(role.startTime).format("LL") }}</h4>
```

Comment:
Consider using another element, e.g. `div`, with styled text size to avoid screen reader/accessibility issues:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements

Reply:
Accounted for.